jetson/percep/camera.cpp

```diff
@@ -414,9 +434,13 @@ void Camera::disk_record_init() {
 
 //Writes point cloud data to data folder specified in build tag 
 void pcl_write(const cv::String &filename, pcl::PointCloud<pcl::PointXYZRGB>::Ptr &p_pcl_point_cloud){
-  std::cout << "name of path is: " << filename << endl;
+  #if PERCEPTION_DEBUG
+    std::cout << "name of path is: " << filename << endl;
+  #endif
   try{ pcl::io::savePCDFileASCII (filename, *p_pcl_point_cloud); }
-  catch (pcl::IOException &e){cerr << e.what();}
+  catch (pcl::IOException &e){
+      cout << e.what();
+    }
```

Comment:
Can you fix the placement of this close brace?

Reply:
Fixed